api/src/services/import-export.ts

```diff
@@ -297,10 +300,27 @@ export class ExportService {
 					schema: this.schema,
 				});
 
+				const usersService = new UsersService({
+					schema: this.schema,
+				});
+
+				const user = await usersService.readOne(this.accountability.user, {
+					fields: ['first_name', 'last_name', 'email'],
+				});
+
+				const message = `
+Hello ${userName(user)},
+
+Your export of ${collection} is ready. <a href="${new Url(env['PUBLIC_URL'])
+					.addPath('admin', 'files', savedFile)
+					.toString()}">Click here to view.</a>
```

Comment:
Wondering if we should put quotes around the collection name or make it italic?
Also I'd probably put the composition of the link outside of `message`.

Reply:
> Wondering if we should put quotes around the collection name or make it italic?

I was wondering about quotes as well! But opted not to add it since the subject doesn't have it:

https://github.com/directus/directus/blob/ab18fb01244f830fd4b896e39adb73777f2885bf/api/src/services/import-export.ts#L322

or should we perhaps have quotes on both the body and the subject for consistency? 

> Also I'd probably put the composition of the link outside of message.

Done 👍 Although slightly unrelated, did the same for the comment message in activity service mentioned in the PR as well for consistency sake.